crypto-ffi/Cargo.toml

```diff
@@ -9,12 +9,16 @@ publish = false
 
 [lib]
 name = "core_crypto_ffi"
-crate-type = ["cdylib", "staticlib"]
```

Comment:
Won't this bloat the binary like the last time we tried ?

Reply:
You're right, I'll revert the change